contracts/base/IEntity.sol

```diff
@@ -1,4 +1,5 @@
 pragma solidity 0.6.12;
+pragma experimental ABIEncoderV2;
```

Comment:
Why is this needed?

Reply:
ABIEncoderV2 enables passing of struct data types into a function from web3 or another contract.
If this is not needed, we can remove.

e.g., 
struct Data {
    string text;
  }

function Insert(Data memory recordData) public returns(bool success) {}